docs/site/static/json/developer/getting-started/create-a-package.json

```diff
@@ -1,29 +1,38 @@
 [
   {
-    "questionText": "What is the purpose of the `key` ability in a struct declaration in Move?",
+    "questionText": "What is the command to create a new IOTA Move package called 'my_first_package'?",
     "answerOptions": [
-      { "answerText": "It allows a struct to be stored and transferred.", "isCorrect": true },
```

Comment:
The quizz being replaced was a duplicate? It's already somewhere else? We're not actually removing it?

Reply:
Yes